docs/root/configuration/http_filters/fault_filter.rst

```diff
@@ -36,6 +27,44 @@ Configuration
 * :ref:`v2 API reference <envoy_api_msg_config.filter.http.fault.v2.HTTPFault>`
 * This filter should be configured with the name *envoy.fault*.
 
+.. _config_http_filters_fault_injection_http_header:
+
+HTTP header fault configuration
+-------------------------------
+
+The fault filter has the capability to allow fault configuration to be specified by the caller.
+This is useful in certain scenarios in which it is desired to allow the client to specify its own
+fault configuration. The currently supported header controls are:
+
+* Request delay configuration via the *x-envoy-throttle-request-latency* header. The header value
+  should be an integer that specifies the number of milliseconds to throttle the latency for.
```

Comment:
this seems a bit strange isn't it? throttle and latency .. How about simply x-envoy-fault-delay (the fault here is an root, and others can be added under the x-envoy-fault-..)

Reply:
Sounds good. How about `x-envoy-fault-delay-request` ? To keep with nesting. Will change.